### ISSUE TITLE
facebook on maven official

### ISSUE THREAD

**Issue by axet** [avianey/facebook-api-android-maven#17]
Please update your project page.

http://search.maven.org/#artifactdetails%7Ccom.facebook.android%7Cfacebook-android-sdk%7C3.19.1%7Caar

**Comment by afollestad** [comment]
Yep, it's up to 3.23.1 on Maven Central now. You should tell people this repo is deprecated.
